### PR TITLE
[Keyboard] Add ingenieurrr

### PR DESCRIPTION
**Pull request by Pascal0816**
new file:   keyboards/ingenieurrr/halconf.h
	new file:   keyboards/ingenieurrr/info.json
	new file:   keyboards/ingenieurrr/ingenieurrr.c
	new file:   keyboards/ingenieurrr/keymaps/default/keymap.c
	new file:   keyboards/ingenieurrr/keymaps/default/rules.mk
	new file:   keyboards/ingenieurrr/keymaps/via/keymap.c
	new file:   keyboards/ingenieurrr/keymaps/via/rules.mk
	new file:   keyboards/ingenieurrr/mcuconf.h
	new file:   keyboards/ingenieurrr/rules.mk
	keyboards/ingenieurrr/readme.md

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ x] New feature
- [ ] Enhancement/optimization
- [x ] Keyboard (addition or update)
- [x ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x ] I have added tests to cover my changes.
- [x ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ingenieurrr/config.h [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+#pragma once
+
+#define WS2812_PWM_DRIVER PWMD4
+#define WS2812_PWM_CHANNEL 4
+#define WS2812_DMA_STREAM STM32_DMA1_STREAM6
+#define WS2812_DMA_CHANNEL 2
+#define WS2812_PWM_TARGET_PERIOD 800000
+
+//#define RGB_MATRIX_LED_COUNT 25
```
Comment (waffle87) [inline keyboards/ingenieurrr/config.h:12]:
```suggestion
```

File: keyboards/ingenieurrr/ingenieurrr.c
Comment (waffle87) [inline keyboards/ingenieurrr/ingenieurrr.c:1]: This file should be removed. If you intend to use RGB Matrix, support for it should be added to `info.json`.

File: keyboards/ingenieurrr/keymaps/default/rules.mk
Comment (waffle87) [inline keyboards/ingenieurrr/keymaps/default/rules.mk:1]: This file should be removed. VIA is not to be enabled in the `default` keymap.

File: keyboards/ingenieurrr/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// Copyright 2023 contact@vwolf.be
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    
+    [0] = LAYOUT(
+        KC_ESC,     KC_1,   KC_2,   KC_3,   KC_4,   KC_5,   KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,   KC_BSPC,    RGB_MOD,    RGB_TOG,   // RGB_VAI,  RGB_VAD,  RGB_SPI,
```
Comment (waffle87) [inline keyboards/ingenieurrr/keymaps/default/keymap.c:9]: Commented out keycodes cause compilation errors...

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Comment by dunk2k** [comment]
`readme.md` file missing - template found at `/data/templates/keyboard/readme.md`

**Pull request closed without merging**